idb/postgres/internal/schema/setup_postgres_sql.go

```diff
@@ -121,4 +121,12 @@ CREATE TABLE IF NOT EXISTS account_app (
 
 -- For looking up existing app local states by account
 CREATE INDEX IF NOT EXISTS account_app_by_addr_partial ON account_app(addr) WHERE NOT deleted;
+
+-- For looking up app box storage
+CREATE TABLE IF NOT EXISTS app_box (
```

Comment:
I believe you also need to add a migration step in postgres_migrations.go to create this new table in existing indexers.